mrp/src/mrp/cmd/up.py

```diff
@@ -117,6 +120,12 @@ def cli(
     if not names:
         raise ValueError("No processes found")
 
+    if attach and len(names) != 1:
+        raise ValueError(f"Cannot attach to {len(names)} processes")
```

Comment:
Can we add here that attach expects only 1 process as input.

Reply:
Done